### PR TITLE
Adding support for SWS-001 weather station (sensors only)

### PR DESCRIPTION
**Pull request by sebastianciupinski**
Weather station supports also outside climate, but since it is taken from internet, based on location, there is no need to integrate that.

### CODE REVIEW

File: custom_components/tuya_local/devices/smart_weather_station.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+name: Smart Weather Station
+products:
+  - id: 000001bbfd
```
Comment (make-all) [inline custom_components/tuya_local/devices/smart_weather_station.yaml:3]: Is this the product id?  It looks more like what appears in the modelId field of Query Things Data Model API call, which is something different (maybe some internal id for Tuya cloud).

File: custom_components/tuya_local/devices/smart_weather_station.yaml [hunk 2]
```diff
@@ -0,0 +1,93 @@
+name: Smart Weather Station
```
Comment (make-all) [inline custom_components/tuya_local/devices/smart_weather_station.yaml:1]: HA naming convention: Only capitalise the first word of a name (unless a trademark or abbreviation that requires it)

File: custom_components/tuya_local/devices/smart_weather_station.yaml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+products:
+  - id: 000001bbfd
+    name: Smart Weather Station
+primary_entity:
+  entity: sensor
+  name: local temperature
```
Comment (make-all) [inline custom_components/tuya_local/devices/smart_weather_station.yaml:7]: HA naming convention: capitalise the first word of an entity name.
I'm not sure whether "local" is the right word, it implies maybe a web service used to get the local temperature for your area. Many other devices use "Internal" for built in sensors where they also support external probes, which is probably clearer.

File: custom_components/tuya_local/devices/smart_weather_station.yaml [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    - id: 131
+      name: sensor
+      type: integer
+      unit: C
+      mapping:
+        - scale: 10
+      class: measurement
+secondary_entities:
+  - entity: sensor
+    name: local humidity
```
Comment (make-all) [inline custom_components/tuya_local/devices/smart_weather_station.yaml:19]: Same comments abourt HA naming convention and "local" as above.

File: custom_components/tuya_local/devices/smart_weather_station.yaml [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+      - id: 133
+        type: integer
+        name: sensor
+        unit: C
+        mapping:
+          - scale: 10
+        class: measurement
+        
+  - entity: sensor
+    name: sensor 1 humidity
```
Comment (make-all) [inline custom_components/tuya_local/devices/smart_weather_station.yaml:41]: HA naming convention. Also the rest of the entities below.

File: custom_components/tuya_local/devices/smart_weather_station.yaml (hunk 2 above)
Comment (sebastianciupinski) [reply custom_components/tuya_local/devices/smart_weather_station.yaml:1]: This is how device introduces itself in tuya app, but I am ok with changing it.

File: custom_components/tuya_local/devices/smart_weather_station.yaml (hunk 3 above)
Comment (sebastianciupinski) [reply custom_components/tuya_local/devices/smart_weather_station.yaml:7]: This is how sensor is named in app. I also do not get naming convention. I mean I see plendy of devices having without first letter capitalized.

File: custom_components/tuya_local/devices/smart_weather_station.yaml (hunk 1 above)
Comment (sebastianciupinski) [reply custom_components/tuya_local/devices/smart_weather_station.yaml:3]: it is going to be changed to "fpgistthfiweyaqr"

File: custom_components/tuya_local/devices/smart_weather_station.yaml (hunk 3 above)
Comment (make-all) [reply custom_components/tuya_local/devices/smart_weather_station.yaml:7]: The convention is relatively recent, you may be looking at older integrations that are not yet updated.

https://developers.home-assistant.io/blog/2022/07/10/entity_naming/

**Pull request merged**